docs/user/docs/tools/capella/working-with-git.md

```diff
@@ -0,0 +1,157 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Working with Git
+
+In this guide we explain how to use Git via Eclipse. When using Git we make
+changes _locally_ and push them to a _remote_ (Gitlab or Github mainly)
+repository. Here we explain how to get made changes to the remote repository so
+that collegues can see and review them.
+
+## Clone a Repository
+
+First open the View _Git Repository_ by clicking on the magnifying glass (1) in
+the right corner.
+
+![Step 1: Open View Search](working-with-git/search-view.png)
+
+Searching looks like:
+
+![Step 2: Search for View](working-with-git/search-git-repository-view.png)
+
+When clicking "Clone a Git repository" a view "Source Git Repository" to enter
+the repository url opens. Enter here the URL of the _Remote_ of your repository
+and if it is not a public repository enter your username and password. After
+clicking "Next" the "Branch Selection" opens. Here you can pick a branch on
+which you want to work. After clicking "Next" another time, the view "Local
+Destination" opens. This shows where your work is going to be stored locally.
+
+ <!-- prettier-ignore -->
+
+!!! info Tick the box: "Import all existing Eclipse projects after clone
```

Comment:
```suggestion
<!-- prettier-ignore -->
!!! info
    Tick the box: "Import all existing Eclipse projects after clone
```

![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/0ecc70a2-8943-4ae7-a693-07ee2387cea5)

This doesn't render properly.